_posts/en/newsletters/2020-05-20-newsletter.md

```diff
@@ -0,0 +1,181 @@
+---
+title: 'Bitcoin Optech Newsletter #98'
+permalink: /en/newsletters/2020/05/20/
+name: 2020-05-20-newsletter
+slug: 2020-05-20-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter relays a request for comments on a proposed
+change to the BIP341 taproot transaction digest and briefly summarizes
+discussion about a new and more concise protocol for atomic swaps.  Also
+included are our regular sections describing changes to services and
+client software, new releases and release candidates, and notable
+changes to popular Bitcoin infrastructure software.
+
+## Action items
+
+- **Evaluate proposed changes to BIP341 taproot transaction digest:** as
+  described in [last week's newsletter][news97 spk commit], there has
+  been a request for [taproot][topic taproot] signatures to make an
+  additional commitment to the scriptPubKeys of the all the UTXOs being
+  spent in a transaction.  Anthony Towns has [suggested][towns
+  suggestion] how [BIP341][] might be updated for this change and Pieter
+  Wuille has [asked][wuille rfc] whether anyone has any objections.  If
+  you have any questions or concerns about the proposed transaction
+  digest revision, we suggest either replying to the mailing list or
+  contacting the BIP341 authors directly.
+
+## News
+
+- **Two-transaction cross chain atomic swap or same-chain coinswap:**
+  Ruben Somsen [posted][somsen post] to the Bitcoin-Dev mailing list and
+  [created a video][somsen video] describing a procedure for a trustless
+  exchange of coins using only two transactions, named Succinct Atomic
+  Swaps (SAS).  The [previous standard protocol][nolan swap] uses four
+  transactions.  Somsen's SAS protocol leaves each party holding coins
+  that they can spend at any time---but which they may need to spend on
+  short notice if their counterparty attempts theft (similar to how LN
+  channels need to be monitored).  Additionally the keys necessary to
+  spend the coins won't be contained in the user's [BIP32][] HD wallet,
+  so additional backups may be required.
+
+    Advantages of the protocol are that it requires less block space
+    than existing protocols, it saves on transaction fees (both by using
+    less block space and potentially by requiring less urgency for its
+    settlement transactions), it only requires consensus-enforced
+    timelocks on one of the chains in a cross-chain swap, and that it
+    doesn't depend on any new security assumptions or Bitcoin consensus
+    changes.  If taproot is adopted, it can be used even more privately
```

Comment:
ambiguous "it"; perhaps "this proposal (or protocol) would allow it to be used even more privately..."